From 52f0ffce7c6251b093db807f09089b6aad97cf10 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Wed, 13 Nov 2024 20:12:58 +0100
Subject: [PATCH] Add missing import for gtksourceview

This is needed to call `gtk_source_init()` and `gtk_source_finalize()`.

Fixes: 0ba7f05cba7feaf2ae2c220596aead5dfc676675
---
 src/main.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/main.c b/src/main.c
index eb07a0c..618d117 100644
--- a/src/main.c
+++ b/src/main.c
@@ -20,6 +20,7 @@
 #include <config.h>
 #include <glib.h>
 #include <gio/gio.h>
+#include <gtksourceview/gtksource.h>
 
 #include "application.h"
 
-- 
GitLab

